binr/radare2/radare2.c

```diff
@@ -906,7 +906,7 @@ int main(int argc, char **argv, char **envp) {
 								// PoC -- must move -rk functionalitiy into rcore
 								// this may be used with caution (r2 -nn $FILE)
 								r_core_cmdf (&r, "Sf");
-								r_core_cmdf (&r, ".!rabin2 -rk. '%s'", r.file->desc->name);
+								r_core_cmdf (&r, ".!rabin2 -rk. \"%s\"", r.file->desc->name);
```

Comment:
Using single quotes `'` was making rabin to try to open the file containing `'` in its name.
The double quotes should be fine regardless of the OS.